File: PHP Excel Examples/WorkbookApiDemo/convertWorkbook.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;
use Spire\Cloud\Excel\Sdk\Model\ExportFormat;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorkbookApi($configuration);

$name = "ConvertWorkbook.xlsx";
#Supports formats : Xlsx, Xlsb, Xls, Ods, Pdf, Xps, Ps, Pcl
$format = ExportFormat::XLSX;
$options = null;
$password = null;
$storage = null;
$folder = "input";
$result = $apiInstance->convertWorkbook($name, $format, $options, $password, $storage, $folder);

File: Python Excel Examples/WorkbookApiDemo/convertWorkbook.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.workbook_api.WorkbookApi(configuration)

name = "ConvertWorkbook.xlsx"
password = ""
storage = ""
format = "Xlsx"  # Available format: Xlsx, Xls, Xlsb, Ods, Xps, Ps, Pcl, Pdf
folder = "ExcelDocument"
result = api.convert_workbook(name, format=format,password=password, storage=storage, folder=folder)